challenges/objects-arrays.js

```diff
@@ -43,23 +67,36 @@ const graduates = [{"id":1,"first_name":"Cynde","university":"Missouri Southern
 {"id":10,"first_name":"Hube","university":"Universitat Rovira I Virgili Tarragona","email":"hlethbrig9@foxnews.com"}]
 
 /* Request 1: Create a new array called universities that contains all the univeristies in the graduates array.  
-
 Once you have the new array created, sort the universities alphabetically and log the result. */
+
 const universities = [];
-console.log(universities)
+for(let i = 0; i < graduates.length; i++) {
+  universities.push(graduates[i]['university']);
+}
+universities.sort();
+console.log(universities);
 
 /* Request 2: Create a new array called contactInfo that contains both first name and email of each student. 
-
 The resulting contact information should have a space between the first name and the email information like this: 
 Name email@example.com
-
 Log the result of your new array. */
 const contactInfo = [];
+for(let i = 0; i < graduates.length; i++) {
+  contactInfo.push(`${graduates[i]['first_name']} ${graduates[i]['email']}`); 
+};
+
 console.log(contactInfo);
 
 
 /* Request 3: Find out how many universities have the string "Uni" included in their name. Create a new array called uni that contains them all. Log the result. */
 const uni = [];
+for(let i = 0; i < graduates.length; i++) {
+  let str = graduates[i]['university']; 
+  let regex = RegExp(/uni/gi); 
+  if (regex.test(str) === true) {
+  uni.push(graduates[i]['university']);
+  }
```

Comment:
_test_ works...._match_ would work as well. You can also use the array method _includes_ to solve this problem as well! Good job!